components/cursor/Cursor.tsx

```diff
@@ -0,0 +1,31 @@
+import CursorSVG from "@/app/assets/CursorSVG";
+
+type Props = {
+    color: string;
+    x: number;
+    y: number;
+    message?: string;
+}
+
+const Cursor = ({ color, x, y, message }: Props) => {
+    return (
+        <div
+            className="pointer-events-none absolute top-0 left-0"
+            style={{ transform: `translateX(${x}px) translateY(${y}px)`}}>
+            <CursorSVG color={color} />
```

Comment:
für die Lebarkeit eine Zeile tiefer stellen

```suggestion
            <CursorSVG color={color} 
        />
```